source/core/coverage/proftab.sql

```diff
@@ -1,4 +1,9 @@
-create table plsql_profiler_runs
+declare
```

Comment:
I'm still thinking if we should make scripts rerunable or drop objects on uninstall.
Also, since we modify script provided from Oracle,  should we split it into one script per object?